### ISSUE TITLE
kn service get <service-name> will list all the services

### ISSUE THREAD

**Issue by savitaashture** [knative/client#143]
Get all the services in particular namespace
```
kn service get -n test
NAME           DOMAIN                          GENERATION   AGE    CONDITIONS   READY     REASON
autoscale-go   autoscale-go.test.example.com   1            124m   0 OK / 3     Unknown   RevisionMissing : Configuration "autoscale-go" is waiting for a Revision to become ready.
test1          test1.test.example.com          1            118m   0 OK / 3     False     RevisionMissing : Configuration "test1" does not have any ready Revision.
```

get a particular service by specifying service name

**current:**
```
kn service get -n test autoscale-go
NAME           DOMAIN                          GENERATION   AGE    CONDITIONS   READY     REASON
autoscale-go   autoscale-go.test.example.com   1            125m   0 OK / 3     Unknown   RevisionMissing : Configuration "autoscale-go" is waiting for a Revision to become ready.
test1          test1.test.example.com          1            119m   0 OK / 3     False     RevisionMissing : Configuration "test1" does not have any ready Revision.
```

**Expected:** Should list only that perticular service.
something like below
```
kn service get -n test autoscale-go
NAME           DOMAIN                          GENERATION   AGE    CONDITIONS   READY     REASON
autoscale-go   autoscale-go.test.example.com   1            125m   0 OK / 3     Unknown   RevisionMissing : Configuration "autoscale-go" is waiting for a Revision to become ready.
```

Because of above behavior when we do 

```
kn service get -n test autoscale-go -o yaml
```

It will give the detailed yaml info for all the services.

**Applicable for all kn commands**

**Comment by navidshaikh** [comment]
Resource name as argument to `get` command is not yet implemented, `get` command's default behavior is to list all resources in requested namespace. Single resource's `get` is actually `describe` command in human readable format which should support different output formats with `-o` option. @rhuss is on it check https://github.com/knative/client/pull/75

**Comment by rhuss** [comment]
@navidshaikh I think `get` should be able to list a single resource as well (in short form), much like `kubectl get` does, too. This is especially useful for machine-readable output (`-o json`). We could use the name as literal or as a prefix file (which would be nice for revisions to catch all similar revisions).

`kn service describe` then is for detailed human-readable output only, but `kn get service myservice -o yaml` is important so that `kn` can easily be used for scripting, too.

**Comment by navidshaikh** [comment]
@rhuss : sounds good, +1

> kn get service myservice -o yaml is important so that kn can easily be used for scripting, too.

:+1:

**Comment by savitaashture** [comment]
@rhuss @navidshaikh Thank you 

/assign

**Comment by rhuss** [comment]
Fixed with #150
